components/nds-tabs/src/Tabs.js

```diff
@@ -161,5 +184,9 @@ export class Tabs extends Component {
 }
 
 Tabs.propTypes = {
-	children: PropTypes.oneOfType([PropTypes.arrayOf(Tab), Tab]).isRequired
+	children: PropTypes.oneOfType([PropTypes.arrayOf(Tab), Tab]).isRequired,
+	initialTab: PropTypes.number,
+	changeCallback: PropTypes.func,
+	hash: PropTypes.string,
```

Comment:
Should this be `initialHash` it being a `hash` prop implies you could control it from a calling component which feels a bit weird

Reply:
Oh wait just seen the NOC PR too. I wonder if that logic that's currently living in NOC ie:
```
props.hash = document.location.hash || "";
```
should live in this component? Ie we either decide:

- Hashes control tabs in general
- The hash controls initial tab
- There's a separate initial index prop

Maybe one to talk through as that was a terrible explanation

Reply:
I think my angle at the moment is that the component shouldn't reference `document` but that may be unfounded.

Reply:
We wouldn't want to use `document` would we, in case we're using a custom router?